lib/awesome_rails_console/railtie.rb

```diff
@@ -21,7 +21,7 @@ def use_awesome_print_for_formatting
     end
 
     def set_prompt_name_to_project_name
-      Pry.config.prompt_name = Rails.application.class.parent_name.underscore.dasherize
+      Pry.config.prompt_name = Rails.application.class.module_parent_name.underscore.dasherize
```

Comment:
Hi @Mayurifag, thank you for the PR

I noticed that this change will break the compatibility with Rails <= 5. After a quick research, I guess this will work for both Rails <= 5 and 6

```suggestion
      Pry.config.prompt_name = PryRails::Prompt.project_name
```

ref: https://github.com/rweng/pry-rails/commit/fcae7f157e384d4e27bd35b58d8bf2c941816dd6

I will try to find some time to test and release in few days. Thanks again for your contribution!

Reply:
Hi @ascendbruce thats going to be nice, thanks! :+1: :smile: